### ISSUE TITLE
[maps/hayes_autos_body_shop] Hayes improvements

### ISSUE THREAD

**Issue by SilverChix** [InZidiuZ/legacy-assets-issue-tracker#600]
Name:
Who is suggesting this? Please include discord name.
Sierra Wrecker (Owner of hayes), SilverChix#9157

Summary:
Write a short summary of what your suggestion is about.
Replacing the bay doors with the same door style as Mosley's with the glass bay doors
adding an outside bay to allow bigger trucks to get fixed up.

Link:
If there are any links to what you're suggesting, please add them here.

Reason:
Why would your suggestion be a good addition to Legacy?

Replacing or adjusting the Bay door will help customers to know when employees are there instead of driving by because the doors are closed in their prospective, so by having glass doors it helps the customers see inside the shop and see if anyone is available for a repair . :)
Having an outside bay for the bigger trucks for hayes will allow our customers to be able to come and fix their vehicles that are bigger than the average car.

**Comment by poopgawd** [comment]
I'd like to see this because usually I can't tell if the shop is open or not until I drive right up to the doors

**Comment by antihero6713** [comment]
I am but one guy please be patient

**Comment by SilverChix** [comment]
> I am but one guy please be patient

I do apologize I believe I had messed up on the first one and was trying to remake it, and sorry it my first time using the github my apologies

**Comment by RetroCharles** [comment]
> 

Thanks anti, appreciate all your work and time.

**Comment by RetroCharles** [comment]
> I am but one guy please be patient

We appreciate you anti, thanks for your dedicated time.

**Comment by antihero6713** [comment]
![image](https://user-images.githubusercontent.com/25892037/163316847-85dff358-fbf1-418b-918e-dfd8cace1f75.png)
Doors are dont know about getting it set up for larger vehicle. That will be a lot of work. 
Got rid of the dumpster that cause issues

**Comment by SilverChix** [comment]
omg they look perfect thank you also I am fine with just the glass doors until further notice I appreciate everything you do

**Comment by antihero6713** [comment]
Going to close make a new issue if want other stuff.
